### ISSUE TITLE
ClojureScript support?

### ISSUE THREAD

**Issue by eprozium** [cursive-ide/gradle-clojure#3]
Is supporting ClojureScript in the scope/roadmap of this project, or it would require a separate Gradle Plug-in?

**Comment by cursive-ghost** [comment]
Yes, it's absolutely in scope. I haven't looked at it yet, but it would be a great feature to add.

**Comment by aadrian** [comment]
@cursive-ide any news on this?

Thank you.
